application/models/admin_model.php

```diff
@@ -52,6 +52,21 @@ public function createAccount($user, $password){
 		$answer = new AJAXAnswer("Account successfully created!", true);
 		return $answer->getJSON();
 	}
+	
+	public function updateAccount($user, $password){
+		if(strlen($password) < 6){
+			$answer = new AJAXAnswer("Error: the password must be 6 characters or longer", false);
+			return $answer->getJSON();
+		}
+
+		$salt = substr(md5(uniqid(rand(), true)), 0, 16);
+		$hash = hash("sha512", $password . $salt);
+		$query = $this->database->prepare("UPDATE BAT_web SET password = '$hash', salt = '$salt' WHERE user = '$user'");

```

Comment:
Just going to use Prepared Statement (or whatever it is called in PHP), to avoid sql injections.